jax/_src/lax/lax.py

```diff
@@ -150,12 +152,30 @@ def ceil(x: Array) -> Array:
   r"""Elementwise ceiling: :math:`\left\lceil x \right\rceil`."""
   return ceil_p.bind(x)
 
-def round(x: Array) -> Array:
+class RoundingMethod(Enum):
```

Comment:
I wonder if this should be `IntEnum`. That might remove the need for `RoundingMethodLike`?

Reply:
Thanks! I did not know about `IntEnum`, it indeed seems very appropriate :)